python/lsst/pipe/tasks/functors.py

```diff
@@ -475,10 +484,12 @@ def __call__(self, data, **kwargs):
 
         Parameters
         ----------
-        data : `lsst.daf.butler.DeferredDatasetHandle`,
-               `lsst.pipe.tasks.parquetTable.MultilevelParquetTable`,
-               `lsst.pipe.tasks.parquetTable.ParquetTable`,
-               or `pandas.DataFrame`.
+        data : various
+            The data represented as `lsst.daf.butler.DeferredDatasetHandle`,
+            `lsst.pipe.tasks.parquetTable.MultilevelParquetTable`,
+            `lsst.pipe.tasks.parquetTable.ParquetTable`,
+            `lsst.pipe.base.InMemoryDatasetHandle`,
+            or `pandas.DataFrame`.
```

Comment:
Does this actually accept a DataFrame?  Is that tested?

Reply:
No idea. I was simply adjusting the text.